modules/foreman_provisioning_template.py

```diff
@@ -389,7 +389,7 @@ def main():
         if state == 'present_with_defaults':
             module.fail_json(msg="'state: present_with_defaults' and 'name: *' cannot be used together")
         if state == 'absent':
-            if entity_dict.keys() != ['name']:
+            if list(entity_dict.keys()) != ['name']:
```

Comment:
would `len(entity_dict.keys()) == 1` work, too?
I think we established already that 'name' is in there. It might be better on resources.

Reply:
using `len` would definitely be the cleaner option, but I think you meant `len(entity_dict.keys()) != 1`, didn't you?